### ISSUE TITLE
Planning to fix? 

### ISSUE THREAD

**Issue by nwmiller1** [SwapnikKatkoori/sleeper-ff-bot#24]
seems Like this bot is returning errors for 2021 sleeper. Fixable?

**Comment by SwapnikKatkoori** [comment]
Yup sorry, did not have time to maintain anything this year. It will be fixed for next year.

**Comment by nwmiller1** [comment]
Just wanted to check in on this. Do you think that this bot will be fixed
for this upcoming football season? would love to integrate it into our
slack.

**Comment by sinphoenix** [comment]
ID be willing to help you with the schedules! I have free time i can update it in no time if you like!

**Comment by nwmiller1** [comment]
Looks like this is still throwing error messages. Please let us know if/when updated.
